sources/org.osbuild.ostree

```diff
@@ -114,14 +123,17 @@ class OSTreeSource(sources.SourceService):
     def fetch_one(self, checksum, desc):
         commit = checksum
         remote = desc["remote"]
+        subpaths = remote.get("subpaths") or []
```

Comment:
I'm curious why not `subpaths = remote.get("subpaths", [])` here? Any deeper reason or just preference?

Reply:
_I suck at Python but do not tell my boss!_